--experimental-annotations-ontology is throwing an error

--experimental-annotations-ontology is throwing the error on v0.16.6: Error: Unable to load ontology terms; Syntax error loading OBO ontology

Angela reported in #cellxgene-users

Steps to reproduce: 
`(venv) schambers@CZIMACOS3069 cellxgenedev % make start-server CXG_OPTIONS="--experimental-annotations-ontology --experimental-enable-reembedding --debug" DATASET=./data/krasnow_lab_human_lung_cell_atlas_smartseq2-2-remixed.h5ad
cellxgene launch -p 5005 --experimental-annotations-ontology --experimental-enable-reembedding --debug ./data/krasnow_lab_human_lung_cell_atlas_smartseq2-2-remixed.h5ad
[cellxgene] Starting the CLI...
[cellxgene] in debug mode, setting verbose=True and open_browser=False
[cellxgene] Loading data from krasnow_lab_human_lung_cell_atlas_smartseq2-2-remixed.h5ad, this may take a while...
Error: Unable to load ontology terms
Syntax error loading OBO ontology
make: *** [start-server] Error 1
(venv) schambers@CZIMACOS3069 cellxgenedev % `

--- a/server/common/annotations/annotations.py
+++ b/server/common/annotations/annotations.py
@@ -33,10 +33,10 @@ class Annotations(metaclass=ABCMeta):
             raise OntologyLoadFailure("Unable to find OBO ontology path") from e
 
         except SyntaxError as e:
-            raise OntologyLoadFailure("Syntax error loading OBO ontology") from e
+            raise OntologyLoadFailure(f"{path}:{e.lineno}:{e.offset} OBO syntax error, unable to read ontology") from e
 
         except Exception as e:
-            raise OntologyLoadFailure("Error loading OBO file") from e
+            raise OntologyLoadFailure(f"{path}:Error loading OBO file") from e
 
     def get_schema(self, data_adaptor):
         schema = []

